Project4/styles.css

```diff
@@ -0,0 +1,151 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+$nav-height: 60px;
+h1 h2 h3 {
+  color: #ffc400;
+}
+body {
+  text-align:center;
+  margin:0px;
+  padding:0px;
```

Comment:
By default, the `body` element doesn't have padding, so this is unnecessary:
```suggestion
```
And, even if it were, the same as above would apply: zeros do not need units.